### PR TITLE
fix env variable error in aggregation

### PR DESCRIPTION
**Pull request by JoeyHwong-gk**
Signed-off-by: JoeyHwong <joeyhwong@gknow.cn>

### CODE REVIEW

File: examples/federated_learning/surface_defect_detection/aggregation_worker/aggregate.py [hunk 1]
```diff
@@ -26,8 +26,8 @@ def run_server():
     participants_count = int(Context.get_parameters(
         "participants_count", 1
     ))
-    agg_ip = Context.get_parameters("AGG_IP", "0.0.0.0")
-    agg_port = int(Context.get_parameters("AGG_PORT", "7363"))
+    agg_ip = Context.get_parameters("AGG_BIND_IP", "0.0.0.0")
```
Comment (llhuii) [inline examples/federated_learning/surface_defect_detection/aggregation_worker/aggregate.py:29]: There are two places reading these two environment variables `AGG_BIND_IP`/`AGG_BIND_PORT`, and only one place takes effect. I suggest to keep the one in `lib/sedna/service/server/aggregation.py`, and remove this example place.

Comment (llhuii) [reply examples/federated_learning/surface_defect_detection/aggregation_worker/aggregate.py:29]: Actually the example should not set these values.

Comment (JoeyHwong-gk) [reply examples/federated_learning/surface_defect_detection/aggregation_worker/aggregate.py]: check

**Pull request merged**